pcsx2/Patch_Memory.cpp

```diff
@@ -420,6 +420,23 @@ void _ApplyPatch(IniPatch *p)
 			handle_extended_t(p);
 			break;
 
+		case SHORT_LE_T:
+			if (memRead16(p->addr) != (u16)p->data)
+				memWrite16(p->addr, (u16) SwapEndian(p->data,SHORT_LE_T));
+			break;
+
+		case WORD_LE_T:
+			if (memRead32(p->addr) != (u32)p->data)
+				memWrite32(p->addr, (u32) SwapEndian( p->data , WORD_LE_T ) );
+			break;
+
+		case DOUBLE_LE_T:
+			u64 memle;
```

Comment:
This is gonna bitch about scope probably, so you'll want to put this case in brackets.

Reply:
Looking back it now, I realize that the if statement is probably to check if the memory has already been written so Ill go fix that.
And now I know why I had scope issues with the switch statements, I thought each case was its own scope.

Reply:
Ok I addressed the scope and comparison issue in https://github.com/PCSX2/pcsx2/pull/4255/commits/501f6199cf50a5b004d00d2ed4b87e201965c813